CONTRIBUTING.md

```diff
@@ -133,7 +133,7 @@ ghjk x dev-compose all # or only the envs required (e.g. base prisma s3)
 ghjk x test-e2e # all tests
```

Comment:
Care to add that missing `build-graph` step here?

Reply:
ok, done!